### ISSUE TITLE
Colour of output image is off

### ISSUE THREAD

**Issue by rcbrigg** [deepak112/Keras-SRGAN#36]
I am finding that image generated often appears to be under-saturated. I might be able to remap the output image to match the mean and variance of the input image, but I don't see understand why the network won't be doing this itself. Is this normal or am I missing something?
 
![gan_generated_image_epoch_627](https://user-images.githubusercontent.com/6876945/91086451-dd1d3700-e646-11ea-9f85-0b81746f49c5.png)

**Comment by BassantTolba1234** [comment]
Dear Sir,
Brilliant work , thank you for sharing your code..
Please, according to your GPU version what the time did you take to finish training your network ??
I'm waiting for your fast reply..

**Comment by BassantTolba1234** [comment]
Please all,
I need the code of implementation this part 
the part is 

{The SRResNet networks
were trained with a learning rate of 10−4 and 106 update
iterations. We employed the trained MSE-based SRResNet
network as initialization for the generator when training
the actual GAN to avoid undesired local optima.{
